apitools/base/py/transfer.py

```diff
@@ -344,7 +344,9 @@ def __GetChunk(self, start, end=None, additional_headers=None):
         self.EnsureInitialized()
         end_byte = end
         if self.total_size and end:
-            end_byte = min(end, self.total_size)
+            end_byte = min(end_byte, self.total_size)
+        # if self.chunksize > 0:
+        #     end_byte = min(end_byte, start + self.chunksize)
```

Comment:
I realize that this may also have implications for GetRange, but I like Upload's explicitly named parameter better than a sentinel value of 0 chunksize, precisely to avoid the kinds of issues I mentioned above.

Don't feel strongly about it, though.